app/javascript/react/utils/thresholdEventHandlers.ts

```diff
@@ -178,11 +179,13 @@ export const useThresholdHandlers = (
       }
     };
 
-    const resetThresholds = () => {
-      debouncedHandleInputChange.cancel();
-      setInputValue("");
-      setThresholdValues(initialThresholds);
-    };
+  const resetThresholds = () => {
+    debouncedHandleInputChange.cancel();
+    dispatch(resetToInitialValues());
+    setThresholdValues(initialState);
```

Comment:
This is a tricky one. I didn't find a better solution for this case.  Values that we have from slice in the Threshold component are updated when the user changes the input values. 

This solution is not permanent. It will probably change again in the future once we get the initial values from the backend. Right now they are mocked.